### PR TITLE
[Bug]: Xlsx export issue

### LINKED ISSUE #256: [Bug]: Xlsx export issue with ES8 and Advanced Object Search 5
### Expected behavior

A file to be downloaded.

### Actual behavior

```
Pimcore\Model\DataObject\Service::getCsvData(): Argument #4 ($fields) must be of type array, null given, 
called in /var/www/pimcore/vendor/pimcore/admin-ui-classic-bundle/src/Controller/Admin/DataObject/DataObjectHelperController.php on line 1351
```

Issue can be seen here where `$fields` is `null`.

Screenshot 2024-02-13 at 14 33 35

### Steps to reproduce

Screenshot 2024-02-13 at 14 34 28

1. Save a search.
2. Try to export with Excel from said search.

### PR DESCRIPTION
**Pull request by jtwogw**
Resolves #256

### CODE REVIEW

**Comment by jtwogw** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/Resources/public/js/searchConfig/resultPanel.js [hunk 1]
```diff
@@ -480,7 +480,8 @@ pimcore.bundle.advancedObjectSearch.searchConfig.resultPanel = Class.create(pimc
         var jobs = [];
 
         var fields = this.getGridConfig().columns;
-        var fieldKeys = Object.keys(fields);
+        var fieldKeys = this.fieldsKeys2(fields); // Object.keys(fields);
```
Comment (mcop1) [inline src/Resources/public/js/searchConfig/resultPanel.js]:
```suggestion
        var fieldKeys = this.fieldsKeys2(fields); 
```

File: src/Resources/public/js/searchConfig/resultPanel.js [hunk 2]
```diff
@@ -509,7 +510,8 @@ pimcore.bundle.advancedObjectSearch.searchConfig.resultPanel = Class.create(pimc
                 var rdata = Ext.decode(response.responseText);
 
                 var fields = this.getGridConfig().columns;
-                var fieldKeys = Object.keys(fields);
+				var fieldKeys = this.fieldsKeys2(fields); // Object.keys(fields);
```
Comment (mcop1) [inline src/Resources/public/js/searchConfig/resultPanel.js]:
```suggestion
				var fieldKeys = this.mapRequestParameter(fields); 
```

File: src/Resources/public/js/searchConfig/resultPanel.js [hunk 3]
```diff
@@ -518,6 +520,26 @@ pimcore.bundle.advancedObjectSearch.searchConfig.resultPanel = Class.create(pimc
             }.bind(this)
         });
     },
+	
+	//temp fix 
+	   
+	fieldsKeys2: function(fields )
```
Comment (mcop1) [inline src/Resources/public/js/searchConfig/resultPanel.js]:
```suggestion
   
	mapRequestParameter: function(fields )
```

**Review by mcop1 — COMMENTED** [review]
@jtwogw Thanks for providing a pull request and sorry for the late answer. Could you please have a look at my suggestions. Basically I just removed the comments and tried to think of a better name for method. Feel free to add the suggestions or maybe think of an even better name for the method. 
Thanks again!

File: src/Resources/public/js/searchConfig/resultPanel.js [hunk 4]
```diff
@@ -480,7 +480,8 @@ pimcore.bundle.advancedObjectSearch.searchConfig.resultPanel = Class.create(pimc
         var jobs = [];
 
         var fields = this.getGridConfig().columns;
-        var fieldKeys = Object.keys(fields);
+        var fieldKeys = this.fieldsKeys2(fields);
```
Comment (mcop1) [inline src/Resources/public/js/searchConfig/resultPanel.js]:
```suggestion
        var fieldKeys = this.mapRequestParameter(fields); 
```

**Pull request merged**